test/e2e/tests/swaps/shared.ts

```diff
@@ -25,11 +25,6 @@ type SwapOptions = {
 
 export const buildQuote = async (driver: Driver, options: SwapOptions) => {
   await driver.clickElement('[data-testid="token-overview-button-swap"]');
-  await driver.fill(
-    'input[data-testid="prepare-swap-page-from-token-amount"]',
-    options.amount.toString(),
-  );
-  await driver.delay(veryLargeDelayMs); // Need an extra delay after typing an amount.
```

Comment:
this delay is not needed anymore as we avoid the collision of querying 2 different swaps (the default and the custom token)